tests/test_wave_05.py

```diff
@@ -80,34 +80,34 @@ def test_create_goal(client):
     }
 
 
-@pytest.mark.skip(reason="test to be completed by student")
+#@pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    raise Exception("Complete test")
-    # Act
+    #raise Exception("Complete test")
     # ---- Complete Act Here ----
-
+    response = client.put('/goals/1', json={
+        'title': 'Updated Goal Title'
+    })
+    response_body = response.get_json()
+    
```

Comment:
You could also add an additional call to the database to test that the db.commit() executed too